olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationClient.h

```diff
@@ -270,6 +271,20 @@ class AUTHENTICATION_API AuthenticationClient {
       const std::string& authentication_server_url = kHereAccountProductionUrl,
       size_t token_cache_limit = 100u);
 
+  /**
+   * @brief Creates the `AuthenticationClient` instance.
+   *
+   * @param settings The authentication settings that can be used to configure
+   * the `AuthenticationClient` instance.
+   * @param authentication_server_url The URL of the HERE Account server.
+   * @example "https://account.api.here.com
+   * @param token_cache_limit The maximum number of tokens that can be cached.
+   */
+  AuthenticationClient(
+      AuthenticationSettings settings,
+      const std::string& authentication_server_url = kHereAccountProductionUrl,
```

Comment:
You just added this to the settings, why do we need one more?
Then also probably you need to adapt the AuthenticationSettings value to match this one, means without the `oauth2/token`.
And once you added that there you can also remove the kHereAccountProductionUrl definition from here.